### PR TITLE
Fix example tests

### PR DESCRIPTION
**Pull request by CORDEA**
I noticed broken examples, so I want to fix examples.

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ targets:
       mockito|mockBuilder:
         generate_for:
           - test/end2end/*.dart
+          - example/**.dart
```
Comment (CORDEA) [inline build.yaml:7]: Should I change here? If not, I will remove it.

File: example/example.mocks.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Mocks generated by Mockito 5.0.6 from annotations
```
Comment (CORDEA) [inline example/example.mocks.dart:1]: Should I commit generated files?

File: example/example.dart [hunk 3]
```diff
@@ -35,6 +40,9 @@ void main() {
   });
 
   test("Let's verify some behaviour!", () {
+    // Stub a method before interacting with it.
+    when(cat.sound()).thenReturn('Meow');
```
Comment (CORDEA) [inline example/example.dart:44]: Should I use the `MockCatRelaxed` overall to reduce changes?

File: build.yaml (hunk 1 above)
Comment (srawlins) [reply build.yaml:7]: No that's great. But let's go ahead and alphabetize it, by putting it one line up.

File: example/example.dart (hunk 3 above)
Comment (srawlins) [reply example/example.dart:44]: No I love it. Not-relaxed is the new way! 😁

File: example/example.mocks.dart (hunk 2 above)
Comment (srawlins) [reply example/example.mocks.dart:1]: I have not been checking in the generated test file. You can go ahead and remove this file and I might reconsider later.

File: example/iss/iss_test.mocks.dart [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// Mocks generated by Mockito 5.0.6 from annotations
```
Comment (srawlins) [inline example/iss/iss_test.mocks.dart:1]: Same here; can remove for now.

**Review by srawlins — CHANGES_REQUESTED** [review]
Thanks so much! Just a few small requests.

**Comment by CORDEA** [comment]
Thank you! I tried to fix it, so please check again.

**Review by srawlins — APPROVED** [review]
Sorry, I missed the re-review!

**Pull request merged**